### PR TITLE
Chats controller dto

### PR DESCRIPTION
**Pull request by illia1304**

### CODE REVIEW

File: backend/DTO/ChatControllerDTO/MessageDTO.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace backend.DTO.ChatControllerDTO;
+
+public class MessageDTO
+{
+    public int Id { get; set; }
+    public int ChatId {  get; set; }
+    public string Message { get; set; }
+    public int UserId {  get; set; }
+    public string SentDate {  get; set; }
```
Comment (AlexSabaka) [inline backend/DTO/ChatControllerDTO/MessageDTO.cs:11]: It's better to use DateTime instead of string.

**Review by AlexSabaka — CHANGES_REQUESTED** [review]
Please fix issues that I pointed out in the comments.

**Review by AlexSabaka — CHANGES_REQUESTED** [review]
Please 1. pull changes from main, 2. update missing code.

File: backend/Controllers/ChatsController.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+        var lastMessageInChat = await _chatDbContext.Messages.Where(x => x.ChatId == id).LastOrDefaultAsync(cancellationToken);
+        GetChatDTO chatDTO = new GetChatDTO
+        {
+            Id = chat.Id,
+            ChatParticipant = _mapper.Map<User, GetUserDTO>(chatParticipant),
+            LastMessage = _mapper.Map<Message, MessageDTO>(lastMessageInChat),
+        };
+
+        return Ok(_mapper.Map<Chat, GetChatDTO>(chat));
```
Comment (AlexSabaka) [inline backend/Controllers/ChatsController.cs]: Should return a chatDTO variable

File: backend/Controllers/ChatsController.cs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        {
+            return NotFound(id);
+        }
+
+        if (chat.UserId1 != currentUserId && chat.UserId2 != currentUserId)
+        {
+            return Forbid();
+        }
+
+        int participantId = chat.UserId2 == currentUserId ? chat.UserId2 : chat.UserId1;
```
Comment (AlexSabaka) [inline backend/Controllers/ChatsController.cs]: Should invert the ternary operator result – when currentUserId == UserId2, then participantId = UserId1, and vice versa

**Review by AlexSabaka — CHANGES_REQUESTED** [review]

**Review by Yurii-Shevchuk — APPROVED** [review]
I believe in Oleksiy's judgement and therefore announce this review a decent merger into the main branch

**Pull request merged**